packages/design-system/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "src/types/*",
     "src/types.ts",
     "src/utils/*.ts",
+    "src/utils/*.tsx",
```

Comment:
Why this change if this PR doesn't actually contain any files from the design system?

Reply:
Huh, not sure. I took this PR over, perhaps there was a .tsx file originally that was not needed anymore 🤔 

Will remove this.